src/Chatkit.php

```diff
@@ -347,6 +347,91 @@ public function joinRoom($options)
         return $this->execCurl($ch);
     }
 
+    /**
+     * Add given users to a given room
+     *
+     * @param array $options
+     *                          [Available Options]
+     *                          • user_ids (array|required): Represents the IDs of the users that you want to add to the room.
+     *                          • room_id (integer|required): Represents the room_id with which the room is identified.
+     *
+     * @throws ChatkitException if any required dependencies are missing
+     *
+     * @return array
+     */
+    public function addUsersToRoom($options)
+    {
+        if (!isset($options['room_id'])) {
+            throw new MissingArgumentException('You must provide a Room ID');
+        }
+
+        if (!isset($options['user_ids'])) {
+            throw new MissingArgumentException('You must provide a User ID(s) to add them to a room');
+        }
+
+        $roomId = $options['room_id'];
+
+        $body = [];
+        $body['user_ids'] = $options['user_ids'];
+
+        $token = $this->generateToken([
```

Comment:
When you want a token with the `'su' => true` claim you can instead just call `$this->getServerToken()`